robottelo/ui/puppetclasses.py

```diff
@@ -58,3 +58,66 @@ def delete(self, name, really):
 
         self.delete_entity(name, really, locators["puppetclass.select_name"],
                            locators['puppetclass.delete'])
+
+    def import_scap_client_puppet_classes(self):
+        """Imports puppet-foreman_scap_client puppet classes."""
+        Navigator(self.browser).go_to_puppet_classes()
+        self.wait_for_ajax()
+        self.find_element(locators['puppetclass.import']).click()
+        # Checking if the scap client puppet classes are already imported
+        if self.wait_until_element(
+                locators['puppetclass.environment_default_check']):
+            self.find_element(
+                locators['puppetclass.environment_default_check']
+            ).click()
+            self.find_element(locators['puppetclass.update']).click()
+        else:
+            self.find_element(locators['puppetclass.cancel']).click()
+
+    def update_class_parameter_description(
+            self, class_name=None, parameter_name=None, description=None):
+        """Updates the description field of a given puppet class parameter."""
+        puppet_class = self.search(class_name)
+        if puppet_class is not None:
```

Comment:
Try doing this instead:

```python
if puppet_class is None:
    raise UINoSuchElementError(…)
…
```

If you check for "success" conditions in your code, you'll often find yourself writing code that is unnecessarily indented and hard to read, and that makes adding more checks ard to do. In contrast, if you check for "failure" conditions in your code, you'll find yourself writing code that is less intented, easier to read, and easy to extend with more checks.